modules/ROOT/pages/sync-function.adoc

```diff
@@ -33,6 +33,7 @@ function (doc, oldDoc) {
     // Your code here
 }
 ----
+The Sync Function must be written with backticks (+`+) in the configuration file (xref::getting-started.adoc#configure-sync-gateway[one example]).
```

Comment:
>must

They can also be done with standard JSON double quotes, but in that case must have new lines escaped.